src/js/routes/Register.jsx

```diff
@@ -3,10 +3,8 @@ import PropTypes from 'prop-types';
 import Helmet from 'react-helmet';
 import styled from 'styled-components';
 import { withStyles } from '@material-ui/core/styles';
-import Button from '@material-ui/core/Button';
-import FormControl from '@material-ui/core/FormControl';
-import InputLabel from '@material-ui/core/InputLabel';
-import Select from '@material-ui/core/Select';
+import { FormControl, InputLabel, Select, Button, Card, CardContent } from '@material-ui/core';
```

Comment:
Hi @jarodpeachey, part of our bundle optimization includes splitting up how we import individual packages from material-ui/core: https://material-ui.com/guides/minimizing-bundle-size/
I'll approve this now and roll-back the imports. I just wanted to let you know about this new way for moving forward.